.github/workflows/version.yml

```diff
@@ -5,8 +5,13 @@ on:
     tags:
       - '*.*.*'
 
+permissions: {}
 jobs:
   build:
+    permissions:
+      contents: write # to create branch (peter-evans/create-pull-request)
```

Comment:
This might not need the permissions since the token used here is the Github App Token from an app and not the default Github token.

Reply:
Sorry for missing that. You are right. Fixed.